Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM apeworx/silverback:latest
+
+WORKDIR /app
+COPY ./bots/* ./bots/
+COPY requirements.txt .
+COPY ape-config.yaml .
+RUN pip install --upgrade pip \
+  && pip install -r requirements.txt
+RUN ape plugins install .
+
+CMD ["run", "bots.hollaback:app"]
```

Comment:
in the silverback cluster, this would require running `silverback worker ...` to set it up properly with the cluster

Reply:
currently just testing this to get it to run on Chris' machine at the moment. I will fix this before we merge